source/_integrations/tuya.markdown

```diff
@@ -25,79 +24,123 @@ ha_platforms:
-you know that it is possible to use lower values.
+**2.** Click **Link Tuya App Account** > **Add App Account**.
+![](../images/integrations/tuya/image_004.png)
 
-- **Query device polling interval** (default=120): this option is available only if you have devices that can use the `API query method`. 
-It defines the interval between 2 consecutive calls to the `API query method`, that is used to get the status for a specific device. 
-This method is always used when it is available only one device that can use it. If you set interval value too low, Tuya API will return errors 
-so it is suggested to use the default value until you know that is possible to use lower values.
+**3.** Scan the QR code that appears using the **Tuya Smart** app or **Smart Life** app.
+![](../images/integrations/tuya/image_005.png)
 
-- **Device that will use the query method**: this option is available only if you have devices that can use the `API query method`. 
-Because it is not possible to make multiple calls to the `API query method`. If you have more than one device that can use it you can choose which one will use. This will give a better status refresh for this specific device.
+**4.** Click **Confirm** on your **Tuya Smart** or **Smart Life** app.
 
-- **Device to configure (multi-select list)**: this option is available only if you have a `light` or `climate` device. Selecting a device to 
-configure to options page related to the device will be opened. You can also select more than one devices to configure them simultaneously, 
-but all selected devices must be of the same type.
+**5.** Navigate to the **Device List** tab. You can find the devices that have been added through your **Tuya Smart** or **Smart Life** account.   
+![](../images/integrations/tuya/image_006.png)
 
-### Light Options
+</details>
 
-- **Force color support**: when checked force `color support` for devices that do not report this feature.
+### Install the Tuya Home Assistant integration
 
-- **Brightness range**: change the `brightness range` used for the device. Possible options are:
-    - range 1-255 (default)
-    - range 10-1000
+#### Install Home Assistant
 
-- **Min color temperature**: set minimum `color temperature` expressed in `kelvin` accepted by the light.
+Please refer to the [Home Assistant Official Installation](https://www.home-assistant.io/installation/) documentation to install **Home Assistant Core**.
 
-- **Max color temperature**: set maximum `color temperature` expressed in `kelvin` accepted by the light.
+You can also get help from [Set up Home Assistant Development Environment on Raspberry Pi](https://developer.tuya.com/en/demo/setuphomeassistantdevenv/?_source=b1441bb02314be3e594a0448891aac58) in Tuya Developer Demo Center.
 
-- **Max color temperature reported**: set the maximum `color temperature` value reported by the light.
+#### Install the Tuya integration
```

Comment:
This whole section needs to be replaced with our config flow instructions and can be fully replace by:

`{% include integrations/config_flow.md %}`